src/dnd-list/types.js

```diff
@@ -79,4 +81,6 @@ export type SharedStylePropsArgT = {
   $isRemovable: boolean,
   $isRemovableByMove: boolean,
   $isOutOfBounds: boolean,
+  // $FlowFixMe
+  $value: string,
```

Comment:
why the suppression here?

Reply:
fixed, update to to `React.Node`, flow was being annoying when doing this

```js
`${$value} label`
```

needs to be changed to

```js
`${typeof $value === 'string' ? $value : ''} label`
```

since it can't coerce `React.Node` subtypes to string